dat/Cave Story.dat

```diff
@@ -0,0 +1,18 @@
+clrmamepro (
+	name "Cave Story"
+	description "Cave Story"
+	version 20160108
+	comment "libretro | www.libretro.com"
+)
+
+game (
+	name "Cave Story (English) (1.0.0.6)"
```

Comment:
Rather than the language, should we use the region? To match the naming convention of No-Intro?